tlg-base-config/README.md

```diff
@@ -0,0 +1,28 @@
+# Create the TLG Base Config in Azure (DC1, APP1, CLIENT1)
```

Comment:
Test Lab Guide, per naming by Joe Davies who is creating the Azure documentation series that intends to leverage the template.  Other TLG templates may follow (as-needed) for the documentation series.